pimd/pim_cmd.c

```diff
@@ -6596,82 +6510,6 @@ DEFUN (show_ip_ssmpingd,
 	return CMD_SUCCESS;
 }
 
-static int pim_rp_cmd_worker(struct pim_instance *pim, struct vty *vty,
-			     const char *rp, const char *group,
-			     const char *plist)
-{
-	int result;
-
-	result = pim_rp_new_config(pim, rp, group, plist);
-
-	if (result == PIM_GROUP_BAD_ADDR_MASK_COMBO) {
```

Comment:
This is no longer checked:
old:
eva(config)# ip pim rp 4.5.6.7 225.1.1.0/16
% Inconsistent address and mask: 225.1.1.0/16

new code:
eva(config)# ip pim rp 4.5.6.7 225.1.1.0/16
eva(config)# do show run
Building configuration...
ip pim rp 4.5.6.7 225.1.1.0/16